microsetta_public_api/api/microsetta_public_api.yml

```diff
@@ -619,6 +619,555 @@ paths:
         '404':
           $ref: '#/components/responses/404NotFound'
 
+  '/{dataset}/plotting/diversity/alpha/{alpha_metric}/percentiles-plot':
```

Comment:
if `dataset` is now upfront, should the existing endpoints be removed and instead require the interactions to denote the dataset being worked on? "/{dataset}/plotting/..." seems very similar to "/plotting/...", for example.

Reply:
Could get weird if you name a dataset `'plotting'`. Probably solved by the `/dataset/{dataset}` solution you proposed?

Would rather make two passes to add the new endpoints, then remove the old ones.

Reply:
:+1: